code/modules/mob/living/simple_animal/abnormality/_tools/zayin/wishwell.dm

```diff
@@ -297,41 +309,44 @@
 	if(gift)
 		Dispense(gift)
 
-/obj/structure/toolabnormality/wishwell/proc/Dispense(atom/dispenseobject)
+/obj/structure/toolabnormality/wishwell/proc/Dispense(atom/output_object)
 	playsound(src, 'sound/abnormalities/bloodbath/Bloodbath_EyeOn.ogg', 80, FALSE, -3)
 	var/turf/dispense_turf = get_step(src, pick(1,2,4,5,6,8,9,10))
 	if(!isopenturf(dispense_turf))
 		dispense_turf = get_turf(src)
-	new dispenseobject(dispense_turf)
+
+	new output_object(dispense_turf)
 	var/list/water_area = range(1, dispense_turf)
 	for(var/turf/open/O in water_area)
 		new /obj/effect/particle_effect/water(O)
+
 	visible_message(span_notice("Something comes out of the well!"))
 
 //Throw yourself into the well : The Code
 /obj/structure/toolabnormality/wishwell/user_buckle_mob(mob/living/M, mob/user, check_loc = TRUE)
-	if (!istype(M, /mob/living/carbon/human))
+	if(!istype(M, /mob/living/carbon/human))
 		to_chat(usr, span_warning("It doesn't look like I can't quite fit in."))
 		return FALSE // Can only extract from humans.
 
-	if(M != user)
-		to_chat(user, span_warning("You start pulling [M] into the well."))
-		if(do_after(user, 7 SECONDS, target = M)) //If you're going to throw someone else, they have to be dead first.
-			if(M.stat == DEAD)
-				to_chat(user, span_notice("You throw [M] in the well!"))
-				buckle_mob(M, check_loc = check_loc)
-			else
-				to_chat(user, span_warning("How could you be so cruel? [M] is still alive!"))
-		return
+	if(M == user)
```

Comment:
Does this file change the functioning of wishwell or is it just formatting correction?

Reply:
Actually, yes it does change it very minorly.
Whilst its mostly a formatting change i moved the `if(M.stat != DEAD) {return}` check from the do_after to be in front of the do_after. So instead of trying to stuff a living person for 7 seconds into the well and failing, you get told instantly that they are alive and you should probably start doing CPR you blind fool

Reply:
Okay thats a thumbs up from me.